src/Diary.php

```diff
@@ -26,6 +26,7 @@ public function initRoutes()
 		// Authorization routes
 		Route::add('/login', function() {(new LoginPage())->render(); }, array('get', 'post'));
 		Route::add('/register', function() {(new RegisterPage())->render(); },  array('get', 'post'));
+		Route::add('/diary-entry', function() {(new DiaryEntryPage())->render(); },  array('get', 'post'));
```

Comment:
Considering we don't have much functionality, we could probably rename this to `/view`? Just for the sake of simplicity.

Reply:
I'll do it

Reply:
Done at commit https://github.com/IBZTI2018/TI3-WT-SA/commit/bc79845f8b828e330f33616d560653caf7f73fd8